.github/workflows/ci.yaml

```diff
@@ -15,10 +15,10 @@ jobs:
       steps:
         - name: Checkout
           uses: actions/checkout@v2
-        - name: Setup .NET SDK v6.0.x
+        - name: Setup .NET SDK v7.0.x
```

Comment:
In Consul we use `dotnet-format` tool (https://github.com/G-Research/consuldotnet/blob/309ab12d7be56b6613a3844955c89955254adf71/.github/workflows/ci.yml#L13-L27) because `dotnet format` command doesn't work in net7. (https://github.com/dotnet/format/issues/1500, https://github.com/dotnet/format/issues/1560)